### DISCUSSION TITLE
Bisakah subset sebuah kata dari data character di R

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#441] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114885734/200097415-93dde0f1-3e50-4c08-8734-2ca70e1fe84b.png)

Halo, mau bertanya, 
di R, bisakah kita mengelompokkan data dengan mengambil keyword 1 kata dari data character. Misalnya seperti di ss dataset terlampir, Saya mau buat kolom baru bernama kategori berisi factor: coffee, chocolate, tea. dengan cara memilih product name yang mengandung kata tersebut. 

Terima kasih.

**Comment by kevbow** [comment]
Dear @19807311 ,

Pertanyaan Anda sudah kami baca dan sedang kami usahakan untuk solusinya. Harap menunggu, terima kasih :)

**Comment by dwiknrd** [comment, marked as answer]
Dear [19807311](https://github.com/19807311)

Apabila dilihat dari data yang dilampirkan, untuk mengelompokkan kedalam suatu kelas dari deskripsi product dapat menggunakan algoritma machine learning.

Namun apabila kebutuhannya ingin memanipulasi data character seperti mengambil character terakhir atau diawal dapat menggunakan fungsi yang ada pada package [stringr](https://stringr.tidyverse.org/). Fungsi dapat menyesuaikan sesuai kebutuhan yang terlampir pada cheatsheet berikut https://github.com/rstudio/cheatsheets/blob/main/strings.pdf

Semoga bermanfaat :)

**Reply by 19807311** [reply]
Baik, nanti saya coba pelajari packagenya.
Kalau algoritma machine learning, masih proses belajar di kelas :)
Terima kasih Mas Kevin dan Mbak Dwi.
